tests/src/resultcollection.hh

```diff
@@ -172,5 +159,20 @@ inline auto linearStressResultsOfTetrahedron = []<typename NOP, typename FE>(NOP
   auto feRequirements =
       typename FE::FERequirementType().insertGlobalSolution(Ikarus::FESolutions::displacement, displacement);
 
-  return std::make_tuple(feRequirements, expectedStress, getVertexPositions(fe));
+  return std::make_tuple(feRequirements, expectedStress, Ikarus::utils::getVertexPositions(fe));
 };
+
+template <typename CompileTimeMatrix>
+auto stressResultsToMatrix(const CompileTimeMatrix& expectedResults) {
+  constexpr int vertices   = CompileTimeMatrix::CompileTimeTraits::RowsAtCompileTime;
+  constexpr int voigtComps = CompileTimeMatrix::CompileTimeTraits::ColsAtCompileTime;
+  constexpr int matrixSize = (-1 + Ikarus::ct_sqrt(1 + 8 * vertices)) / 2;
```

Comment:
This line seems strange. Why is the number of vertices  used here so complicated. The number of vertices os a compile time constant therfore I assume this is restricted to one geometry type.

Reply:
This is just bad wording, The `vertices` are the amount of rows, and as I supply the results as row Vectors, the amount of vertices therefore is deduced from the matrix und therefore its generally usable
This function is just used to transform the results in the `resultCollection.hh`